AspNetCore.ReCaptcha/ReCaptchaTagHelper.cs

```diff
@@ -44,10 +44,16 @@ public override void Process(TagHelperContext context, TagHelperOutput output)
             if (Id.ToLower() == "submit")
                 throw new ArgumentException("id can't be named submit");
 
-            Language ??= ViewContext.HttpContext.Features.Get<IRequestCultureFeature>()?.RequestCulture.UICulture.TwoLetterISOLanguageName;
-
             var settings = ViewContext.HttpContext.RequestServices.GetRequiredService<IOptions<ReCaptchaSettings>>().Value;
```

Comment:
If we are going down the `IOptionsSnapshot` route, please also make use of the `IOptionsSnapshot` here aswell. Otherwise, we will generate a captcha, but just not validate it. If we make it the same, if the `Enabled` boolean will be set to `false`  at runtime it will update for both methods at the same time.

Reply:
@michaelvs97 nice catch — I've address your feedback in a new commit.